game_eggs/steamcmd_servers/starbound/egg-starbound.json

```diff
@@ -24,7 +24,7 @@
+            "script": "#!\/bin\/bash\r\n# steamcmd Base Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\n# Image to install with is 'ubuntu:18.04'\r\napt -y update\r\napt -y --no-install-recommends install curl lib32gcc1 ca-certificates\r\n\r\n## just in case someone removed the defaults.\r\nif [ \"${STEAM_USER}\" == \"\" ]; then\r\n    echo -e \"steam user is not set.\\n\"\r\n    echo -e \"Using anonymous user.\\n\"\r\n    STEAM_USER=anonymous\r\n    STEAM_PASS=\"\"\r\n    STEAM_AUTH=\"\"\r\nelse\r\n    echo -e \"user set to ${STEAM_USER}\"\r\nfi\r\n\r\n## download and install steamcmd\r\ncd \/tmp\r\nmkdir -p \/mnt\/server\/steamcmd\r\ncurl -sSL -o steamcmd.tar.gz https:\/\/steamcdn-a.akamaihd.net\/client\/installer\/steamcmd_linux.tar.gz\r\ntar -xzvf steamcmd.tar.gz -C \/mnt\/server\/steamcmd\r\ncd \/mnt\/server\/steamcmd\r\n\r\n# SteamCMD fails otherwise for some reason, even running as root.\r\n# This is changed at the end of the install process anyways.\r\nchown -R root:root \/mnt\r\nexport HOME=\/mnt\/server\r\n\r\n## install game using steamcmd\r\n.\/steamcmd.sh +login ${STEAM_USER} ${STEAM_PASS} ${STEAM_AUTH} +force_install_dir \/mnt\/server +app_update ${SRCDS_APPID} ${EXTRA_FLAGS} validate +quit ## other flags may be needed depending on install. looking at you cs 1.6\r\n\r\n## set up 32 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so\r\n\r\n## set up 64 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk64\r\ncp -v linux64\/steamclient.so ..\/.steam\/sdk64\/steamclient.so\r\n\r\n## Starbound specific setup.\r\ncd \/mnt\/server\/\r\n\r\n\r\n## edit sbinit config\r\nmv \/mnt\/server\/linux\/* \/mnt\/server\/\r\nrm -rf \/mnt\/server\/linux\r\nsed -i -e 's\/\\.\\.\/\\.\/g' \/mnt\/server\/sbinit.config\r\n\r\n## pull starbound_server.config\r\nmkdir -p \/mnt\/server\/storage\r\ncurl -sSL https:\/\/raw.githubusercontent.com\/parkervcp\/eggs\/master\/game_eggs\/steamcmd_servers\/starbound\/starbound_server.config > \/mnt\/server\/storage\/starbound_server.config\r\n\r\n## use Worksop content\r\nif [ \"${WORKSHOP}\" == \"1\" ]; then\r\n    for dir in \/mnt\/server\/steamapps\/workshop\/content\/211820\/*\/\r\n    do\r\n        # Removes the trailing slash from the full path\r\n        dir=${dir%*\/}\r\n        echo Sym-linking mod ${dir##*\/} files into the mods folder\r\n        INDEX=0\r\n        for pakfile in ${dir}\/*.pak\r\n        do\r\n            echo $pakfile\r\n            ln -r -s $pakfile \/mnt\/server\/mods\/${dir##*\/}_${INDEX}.pak\r\n            INDEX=$((INDEX+1))\r\n        done\r\n    done\r\nfi",
             "container": "debian:buster-slim",
```

Comment:
use the Debian Installer image from GHCR

Reply:
> use the Debian Installer image from GHCR

Are you refering to the `ghcr.io/parkervcp/games:source` image that @QuintenQVD0 mentioned? I'm not sure what image I should use as that one doesn't mention debian

Reply:
That, or @parkervcp could try changing it after testing my changes maybe?

Reply:
@Wazbat the install image should the GHCR debian install image. And the game image you should try the 1 I posted and if it works you should use that one

Reply:
> That, or @parkervcp could try changing it after testing my changes maybe?

i pushed a PR to your fork

Reply:
Gotcha, just gonna leave this link here for reference

Wazbat/eggs#1

Reply:
Ahhh I see what you mean now. I was a bit confused why your code review was on this line. I saw the diff and I thought you were refering to the `script` property, not the `container`